src/patterns/components/dictionary/base.hbs

```diff
@@ -0,0 +1,33 @@
+---
+order: 1
+notes: This is the default use of this pattern.
+---
+
+<div class="sprk-c-Dictionary">
+  <dl class="sprk-c-Dictionary__keyvaluepairs">
+    <div class="sprk-c-Dictionary__keyvaluepair">
+      <dt class="sprk-c-Dictionary__key sprk-b-Label">Email Address</dt>
+      <dd class="sprk-c-Dictionary__value">bobwise@quickenloans.com</dd>
```

Comment:
Let's change this email to the spark email or something

Reply:
Good call. 😂